### DISCUSSION TITLE
各大镜像站配置文件生成

### DISCUSSION THREAD

**Discussion by 2096779623** [FloatTech/ZeroBot-Plugin#184] [Ideas]
- 功能名：镜像站配置文件生成mirrorgen
- 功能具体：发送mirrorgen进行各大镜像站配置文件的生成，或者返回帮助页面，比如：mirrorgen -l (linux) -m (镜像站)

**Comment by fumiama** [comment]
适用范围略显狭窄，暂不考虑增加此功能，不过有意愿的开发者可以考虑pr。
